typings/flowdock.d.ts

```diff
@@ -10,5 +10,9 @@ declare module 'flowdock' {
 		public stream(ids: string[]): FlowdockStream;
 		public on(event: string, callback: (error: Error) => void): void;
 		public removeListener(event: string, callback: (error: Error) => void): void;
+		public _request(
```

Comment:
Well, this is actually just a typing around someone else's SDK, which reveals a slightly different error of using someone else's 'private by convention' methods, which I'm fixing.